src/geometry/manifold/times_cont_mdiff.lean

```diff
@@ -0,0 +1,633 @@
+/-
+Copyright (c) 2020 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+
+import geometry.manifold.mfderiv
+
+/-!
+# Smooth functions between smooth manifolds
+
+We define `Cⁿ` functions between smooth manifolds, as functions which are `Cⁿ` in charts, and prove
+basic properties of these notions.
+
+## Main definitions and statements
+
+Let `M ` and `M'` be two smooth manifolds, with respect to model with corners `I` and `I'`. Let
+`f : M → M'`.
+
+* `times_cont_mdiff_on I I' n f s` states that the function `f` is `Cⁿ` on the set `s`
+* `times_cont_mdiff I I' n f` states that the function `f` is `Cⁿ`.
+* `times_cont_mdiff_on.comp` gives the invariance of the `Cⁿ` property under composition
+* `times_cont_mdiff_on.times_cont_mdiff_on_bundle_mfderiv_within` states that the bundled derivative
```

Comment:
What does bundled derivative means? This is clear when reading the file, but not at this point. Why not saying tangent map?